README.md

```diff
@@ -32,6 +32,10 @@ See the documentation below for getting setup in each tool:
 - [component-library](/packages/component-library/README.md#getting-started)
 - [form-schema](/packages/form-schema/README.md#getting-started)
 
+Some of this information is also available in the component's Storybooks:
```

Comment:
I find "this information" confusing (ie what info is included in "this"?)

Reply:
Good point, rewrote some of this with 13c8160!